src/code_190B20.c

```diff
@@ -1,5 +1,454 @@
+    Debuff_THINKING, PlayerAnim_THINKING,
+    Debuff_WEARY, 0x0001000D,
+    Debuff_END,
+};
+
+s32 bMarioDefendAnims[] = {
+    Debuff_NORMAL, PlayerAnim_CROUCH,
+    Debuff_DANGER, PlayerAnim_PANTING,
+    Debuff_STONE, 0x00050001,
+    Debuff_SLEEP, 0x00030004,
+    Debuff_DIZZY, 0x00030006,
+    Debuff_STOP, PlayerAnim_STAND_STILL,
+    Debuff_HUSTLE, PlayerAnim_CROUCH,
+    Debuff_BERSERK, PlayerAnim_CROUCH,
+    Debuff_PARALYZE, 0x00030004,
+    Debuff_FROZEN, 0x0001000E,
+    0xE, 0x00010002,
+    Debuff_TURN_DONE, PlayerAnim_STAND_STILL,
+    0x13, 0x00040029,
+    0x14, 0x0004002A,
+    0x15, 0x00030003,
+    0x18, 0x00030005,
+    0x16, 0x0010001,
+    Debuff_THINKING, PlayerAnim_THINKING,
+    Debuff_WEARY, 0x0001000D,
+    Debuff_END,
+};
+
+s32 bMarioHideAnims[] = {
+    Debuff_NORMAL, PlayerAnim_CROUCH,
+    Debuff_DANGER, PlayerAnim_PANTING,
+    Debuff_STONE, 0x00050001,
+    Debuff_SLEEP, 0x00030004,
+    Debuff_DIZZY, 0x00030006,
+    Debuff_STOP, PlayerAnim_STAND_STILL,
+    Debuff_HUSTLE, 0x0008000E,
+    Debuff_BERSERK, 0x0008000E,
+    Debuff_PARALYZE, 0x00030004,
+    Debuff_FROZEN, 0x0001000E,
+    0xE, 0x00010002,
+    Debuff_TURN_DONE, PlayerAnim_CROUCH,
+    0x13, 0x00040029,
+    0x14, 0x0004002A,
+    0x15, 0x00030003,
+    0x18, 0x00030005,
+    0x16, 0x0010001,
+    Debuff_THINKING, PlayerAnim_THINKING,
+    Debuff_WEARY, 0x0001000D,
+    Debuff_END,
+};
+
+s32 bPeachIdleAnims[] = {
+    Debuff_NORMAL, 0x000A0002,
+    Debuff_TURN_DONE, 0x000C0028,
+    Debuff_END,
+};
+
+s32 bMarioDefenseTable[] = {
+    Element_NORMAL, 0,
+    Element_END,
+};
+
+s32 bPlayerStatusTable[] = {
+    Debuff_NORMAL, 100,
+    Debuff_DEFAULT, 100,
+    Debuff_SLEEP, 100,
+    Debuff_POISON, 100,
+    Debuff_FROZEN, 100,
+    Debuff_DIZZY, 100,
+    Debuff_FEAR, 100,
+    Debuff_STATIC, 100,
+    Debuff_PARALYZE, 100,
+    Debuff_SHRINK, 100,
+    Debuff_STOP, 100,
+    Debuff_DEFAULT_TURN_MOD, 0,
+    Debuff_SLEEP_TURN_MOD, 0,
+    Debuff_POISON_TURN_MOD, 0,
+    Debuff_FROZEN_TURN_MOD, 0,
+    Debuff_DIZZY_TURN_MOD, 0,
+    Debuff_FEAR_TURN_MOD, 0,
+    Debuff_STATIC_TURN_MOD, 0,
+    Debuff_PARALYZE_TURN_MOD, 0,
+    Debuff_SHRINK_TURN_MOD, 0,
+    Debuff_STOP_TURN_MOD, 0,
+    Debuff_END,
+};
+
+ActorDesc bPlayerActorDesc = {
+    .type = 110,
+    .maxHP = 99,
+    .statusTable = bPlayerStatusTable,
+
+    .escapeChance = 0,
+    .airLiftChance = 0,
+    .spookChance = 0,
+    .baseStatusChance = 0,
+    .upAndAwayChance = 0,
+    .spinSmashReq = 4,
+    .powerBounceChance = 80,
+
+    .size = { 33, 43 },
+    .hpBarOffset = { 0, 0 },
+    .statusIconOffset = { -10, 30 },
+    .statusMessageOffset = { 10, 30 },
+};
+
+ActorPartDesc bMarioParts[] = {
+    {
+        .index = 1,
+        .posOffset = { 0, 0, 0 },
+        .targetOffset = { 0, 32 },
+        .opacity = 0xFF,
+        .idleAnimations = bMarioIdleAnims,
+        .defenseTable = bMarioDefenseTable,
+    },
+};
+
+// TODO: split 0x8028358C
+s32 D_80283524[] = {
```

Comment:
hmm, we should at least add the pointers in here to undefined_syms.txt, unless there's some reason not to?